src/GlobalStyles.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from "styled-components";
+
+export default createGlobalStyle`
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 16px;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
```

Comment:
I think you might have forgotten to add the relevant stylesheet to your index.html as well